packages/fether-react/src/Send/SignedTxSummary/SignedTxSummary.js

```diff
@@ -78,22 +78,21 @@ class SignedTxSummary extends Component {
                     render={({ handleSubmit, values }) => (
                       <form className='send-form' onSubmit={handleSubmit}>
                         <fieldset className='form_fields'>
-                          <Field
-                            className='form_field_amount'
-                            disabled
-                            label='Amount'
-                            name='amount'
-                            render={FetherForm.Field}
-                            type='number'
-                          />
-
-                          <Field
+                          <FetherForm.Field
```

Comment:
We need to use Field from `final-form` here, with the correct render prop

Reply:
@amaurymartiny I've pushed a commit that addresses the issue. I also changed it from `defaultValue` to `value` to overcome a warning